README.md

```diff
@@ -2071,33 +2073,226 @@ ctx.dataSource.portNumber=1433
+  "io.getquill" %% "quill-jdbc-monix" % "3.0.0-SNAPSHOT"
+)
+```
+
+#### context definition
+```scala
+lazy val ctx = new MysqlMonixJdbcContext(SnakeCase, "ctx", Runner.default)
+```
+
+#### application.properties
+```
+ctx.dataSourceClassName=com.mysql.jdbc.jdbc2.optional.MysqlDataSource
+ctx.dataSource.url=jdbc:mysql://host/database
+ctx.dataSource.user=root
+ctx.dataSource.password=root
+ctx.dataSource.cachePrepStmts=true
+ctx.dataSource.prepStmtCacheSize=250
+ctx.dataSource.prepStmtCacheSqlLimit=2048
+ctx.connectionTimeout=30000
+```
+
+### Postgres (quill-jdbc-monix)
+
+#### sbt dependencies
+```
+libraryDependencies ++= Seq(
+  "org.postgresql" % "postgresql" % "9.4.1208",
+  "io.getquill" %% "quill-jdbc-monix" % "3.0.0-SNAPSHOT"
+)
+```
 
+#### context definition
+```scala
+lazy val ctx = new PostgresMonixJdbcContext(SnakeCase, "ctx", Runner.default)
 ```
+
+#### application.properties
+```
+ctx.dataSourceClassName=org.postgresql.ds.PGSimpleDataSource
+ctx.dataSource.user=root
+ctx.dataSource.password=root
+ctx.dataSource.databaseName=database
+ctx.dataSource.portNumber=5432
+ctx.dataSource.serverName=host
+ctx.connectionTimeout=30000
+```
+
+### Sqlite (quill-jdbc-monix)
+
+#### sbt dependencies
+```
+libraryDependencies ++= Seq(
+  "org.xerial" % "sqlite-jdbc" % "3.18.0",
+  "io.getquill" %% "quill-jdbc-monix" % "3.0.0-SNAPSHOT"
+)
+```
+
+#### context definition
+```scala
+lazy val ctx = new SqliteMonixJdbcContext(SnakeCase, "ctx", Runner.default)
+```
+
+#### application.properties
+```
+ctx.driverClassName=org.sqlite.JDBC
+ctx.jdbcUrl=jdbc:sqlite:/path/to/db/file.db
+```
+
+### H2 (quill-jdbc-monix)
+
+#### sbt dependencies
+```
+libraryDependencies ++= Seq(
+  "com.h2database" % "h2" % "1.4.192",
+  "io.getquill" %% "quill-jdbc-monix" % "3.0.0-SNAPSHOT"
+)
+```
+
+#### context definition
+```scala
+lazy val ctx = new H2MonixJdbcContext(SnakeCase, "ctx", Runner.default)
+```
+
+#### application.properties
+```
+ctx.dataSourceClassName=org.h2.jdbcx.JdbcDataSource
+ctx.dataSource.url=jdbc:h2:mem:yourdbname
+ctx.dataSource.user=sa
+```
+
+### SQL Server (quill-jdbc-monix)
+
+#### sbt dependencies
+```
+libraryDependencies ++= Seq(
+  "com.microsoft.sqlserver" % "mssql-jdbc" % "6.1.7.jre8-preview",
+  "io.getquill" %% "quill-jdbc" % "3.0.0-SNAPSHOT"
+)
+```
+
+#### context definition
+```scala
+lazy val ctx = new SqlServerMonixJdbcContext(SnakeCase, "ctx", Runner.default)
+```
+
+#### application.properties
+```
+ctx.dataSourceClassName=com.microsoft.sqlserver.jdbc.SQLServerDataSource
+ctx.dataSource.user=user
+ctx.dataSource.password=YourStrongPassword
+ctx.dataSource.databaseName=database
+ctx.dataSource.portNumber=1433
+ctx.dataSource.serverName=host
+```
+
+## quill-async
+
+The `quill-async` module provides simple async support for MySQL and Postgres databases.
+
+#### transactions
+
+The async module provides transaction support based on a custom implicit execution context:
+
+````scala
```

Comment:
hmm it seems that there's a reason for the additional `?

Reply:
No reason. Changed it.